Index: cockpit-195.12/src/ws/test-handlers.c
===================================================================
--- cockpit-195.12.orig/src/ws/test-handlers.c
+++ cockpit-195.12/src/ws/test-handlers.c
@@ -669,7 +669,7 @@ static const DefaultFixture fixture_stat
   .expect = "HTTP/1.1 200*"
     "Cache-Control: max-age=31556926, public*"
     "#badge*"
-    "url(\"logo.png\");*"
+    "url(\"suse-logo-login.svg\");*"
 };
 
 static const DefaultFixture fixture_host_static = {
@@ -705,7 +705,7 @@ static const DefaultFixture fixture_stat
   .expect = "HTTP/1.1 200*"
     "Cache-Control: max-age=31556926, public*"
     "#badge*"
-    "url(\"logo.png\");*"
+    "url(\"suse-logo-login.svg\");*"
 };
 
 static void
